test/hover-test-cases.json

```diff
@@ -0,0 +1,112 @@
+// 1. defalut
```

Comment:
Planning on using this file?

Reply:
used for test case reference. if you are not interested in it, i will just remove it from here.

Reply:
Then remove it, we don't want any unused code :)

Reply:
Done

Reply:
go ahead and merge :)